content/600-about/00-whats-new-in-prisma-docs.mdx

```diff
@@ -12,6 +12,28 @@ Full release notes for Prisma itself, including breaking changes, are published
 
 </TopBlock>
 
+## May 04 2021
+
+That we refactored and expanded the deployment section, refactored connection management
+
+### Release 🚀
+
+`prisma db push` is now out of preview!
+
+- [2.22.0](https://github.com/prisma/prisma/releases/tag/2.22.0) documentation
+
+### New documentation
+
+#### Highlights 💡
+
```

Comment:
Would move this + User/Post stuff to 'Other documentation' or 'other highlights'